### PR TITLE
fix(fa): Orders staff list

### PR DESCRIPTION
**Pull request by Rikkiabb**
## What
Orders staff list by active and name
Applies different color to inactive users

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: apps/financial-aid/backend/src/app/modules/staff/staff.service.ts [hunk 1]
```diff
@@ -33,6 +34,9 @@ export class StaffService {
       where: {
         municipalityId,
       },
+      order: Sequelize.literal(
+        'CASE WHEN active = true THEN 0 ELSE 1 END, name ASC',
```
Comment (MargretFinnboga) [inline apps/financial-aid/backend/src/app/modules/staff/staff.service.ts:38]: You can also do   order: [['active', 'DESC']] I think 💭

**Review by MargretFinnboga — COMMENTED** [review]
Suggestion 🎉

File: apps/financial-aid/backend/src/app/modules/staff/staff.service.ts (hunk 1 above)
Comment (Rikkiabb) [reply apps/financial-aid/backend/src/app/modules/staff/staff.service.ts:38]: Didn't know that, but do you think it is more readable?

**Review by MargretFinnboga — APPROVED** [review]
Slides 🛷

**Pull request merged**